myproject_sql/models/mymodel.py

```diff
@@ -3,16 +3,27 @@
     Index,
     Integer,
     Text,
+    Unicode,
+    UnicodeText,
+    DateTime
 )
 
 from .meta import Base
 
 
 class MyModel(Base):
     __tablename__ = 'models'
-    id = Column(Integer, primary_key=True)
+    value = Column(Integer, primary_key=True)
     name = Column(Text)
+
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = Column(Integer, primary_key=True)
+    title = Column(Unicode)
+    body = Column(UnicodeText)
     value = Column(Integer)
```

Comment:
of what use is this column on your entry?  Does it serve a purpose?  If not, why do you have it here?